docs/server/source/simple-network-setup.md

```diff
@@ -194,14 +217,37 @@ persistent_peers = "<Member 1 node id>@<Member 1 hostname>:46656,\
 <Member N node id>@<Member N hostname>:46656,"
 ```
 
-Now the Member must run two commands:
-- `bigchaindb start`
-- `tendermint node`
+## Member: Start MongoDB, BigchainDB and Tendermint
+
+To start MongoDB, one uses the command `mongod` but that will run it in the foreground. If you want to run it in the background (so it will continue running after you logout), you can use `mongod --fork --logpath /var/log/mongodb.log`. (You might have to create the `/var/log` directory if it doesn't already exist.)
+
+The _recommended_ approach is to reboot the machine and let the MongoDB startup script start MongoDB. Don't do that yet though…
+
+To start BigchainDB, one uses the command `bigchaindb start` but that will run it in the foreground. If you want to run it in the background (so it will continue running after you logout), you can use `nohup`, `tmux`, or `screen`. For example, `nohup bigchaindb start 2>&1 > bigchaindb.log &`
+
+The _recommended_ approach is to create a startup script for BigchainDB, so it will start right after the boot of the operating system. (As mentioned earlier, MongoDB should already have a startup script.)
+
+To start Tendermint, one uses the command `tendermint node` but that will run it in the foreground. If you want to run it in the background (so it will continue running after you logout), you can use `nohup`, `tmux`, or `screen`. For example, `nohup tendermint node 2>&1 > tendermint.log &`
+
+The _recommended_ approach is to create a startup script for Tendermint, so it will start right after the boot of the operating system.
+
+Note: We'll share some example startup scripts in the future. This document is a work in progress.
+
+If you followed the recommended approach and created startup scripts for BigchainDB and Tendermint, then you can reboot the machine now. MongoDB, BigchainDB and Tendermint should all start.
+
+## How Others Can Access Your Node
+
+If you followed the above instructions, then your node should be publicly-accessible with BigchainDB Root URL `http://hostname:9984` (where hostname is something like `bdb7.canada.vmsareus.net` or `17.122.200.76`). That is, anyone can interact with your node using the [BigchainDB HTTP API](http-client-server-api.html) exposed at that address. The most common way to do that is to use one of the [BigchainDB Drivers](./drivers-clients/index.html).
+
+## Refreshing Your Node
+
+If you want to refresh your node back to a fresh empty state, then your best bet is to terminate it and deploy a new virtual machine, but if that's not an option, then you can:
```

Comment:
Good idea to give additional information on how to reset a node, but trashing a virtual machine to start with an empty state is too much IMO.

What about keeping just the paragraph below? You can mention that if you want to reset your node to join a new network you can also remove the `.tendermint` directory.

Reply:
For people who don't fully-understand what is going on, and where all the data is stored, trashing the virtual machine and starting a new one is a completely reasonable and believable way to get back to a fresh state, so I'm not going to delete that sentence.